docs/user/ppl/functions/datetime.rst

```diff
@@ -1152,6 +1152,81 @@ Example::
     +--------------------------+-----------------------------+
 
 
+UTC_DATE
+-------
+
+Description
+>>>>>>>>>>>
+
+Returns the current UTC date as a value in 'yyyy-MM-dd'.
+
+Return type: DATE
+
+Specification: UTC_DATE() -> DATE
+
+Example::
+
+    >od source=people | eval `UTC_DATE()` = UTC_DATE() | fields `UTC_DATE()`
+    fetched rows / total rows = 1/1
+    +--------------+
+    | UTC_DATE()   |
+    |--------------|
+    | 2022-10-03   |
+    +--------------+
+
+
+
+UTC_TIME
+-------
+
+Description
+>>>>>>>>>>>
+
+Returns the current UTC time as a value in 'HH:mm:ss'.
+
+Return type: TIME
+
+Specification: UTC_TIME() -> TIME
+
+Example::
+
+    >od source=people | eval `UTC_TIME()` = UTC_TIME() | fields `UTC_TIME()`
+    fetched rows / total rows = 1/1
+    +--------------+
+    | UTC_TIME()   |
+    |--------------|
+    | 17:54:27     |
+    +--------------+
+
+
+
+
+UTC_TIMESTAMP
+-------
+
+Description
+>>>>>>>>>>>
+
+Returns the current UTC timestamp as a value in 'yyyy-MM-dd HH:mm:ss'.
```

Comment:
Documentation above says: YYYY-MM-DD
Is it better to be consistent?

Reply:
Good point, other way was following Java standard, not MySQL.

Reply:
Made consistent with MySQL. Addressed in https://github.com/Bit-Quill/opensearch-project-sql/commit/59ef32f37ea15e9f44cae06574c3cb9f27db26e1.